### PR TITLE
Feat/algolia recommend

### PR DESCRIPTION
**Pull request by Baroshem**
Closes #28 

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: src/module.ts [hunk 1]
```diff
@@ -16,12 +16,16 @@ interface ModuleBaseOptions {
...
   lite?: boolean;
   instantSearch?: boolean | { theme: keyof typeof InstantSearchThemes };
+  recommend?: boolean;
 }
 
 declare module '@nuxt/schema' {
   interface PublicRuntimeConfig {
     algolia: ModuleBaseOptions
   }
+  interface NuxtConfig {
```
[the remark this answers is not in the document]
Comment (yassilah) [reply src/module.ts:26]: replace `import AlgoliaModule from '../src/module'` with `import AlgoliaModule from '..'` to resolve the exports fields from package.json and it should work fine :)

[the remark this answers is not in the document]
Comment (Baroshem) [reply src/module.ts]: It works! Thanks!

Do you have any other recommendations regarding the code? :)

File: src/runtime/composables/useAlgoliaRecommend.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+export function useAlgoliaRecommend<T>(): UseAlgoliaRecommend<T> {
+  const nuxtApp = useNuxtApp()
+  const { algolia } = useRuntimeConfig()
+  const isRecommendEnabled = algolia.recommend
+  if (!isRecommendEnabled) {
+    console.error('`[@nuxtjs/algolia]` Cannot call useAlgoliaRecommend composable due to missing `algolia.recommend` option')
+  }
+  const algoliaRecommend: RecommendClient = nuxtApp.$algoliaRecommend
```
[the remark this answers is not in the document]
Comment (Baroshem) [reply src/runtime/composables/useAlgoliaRecommend.ts:24]: $algoliaRecommend will not be defined if isRecommendEnabled will be false. So I dont think that we need to check that additionalty, however I agree with throwing an error instead console logging it. The composable will not work anyway so no point in triggering the code.

[the remark this answers is not in the document]
Comment (Baroshem) [reply src/runtime/composables/useAlgoliaRecommend.ts]: Oh, I see that you have already added the recommendation. Nice! Can you also approve if you think it does not require more work? I would like to merge it today and release probably tomorrow.

**Review by yassilah — APPROVED** [review]

**Pull request merged**